crates/jstz_engine/src/string/mod.rs

```diff
@@ -186,6 +187,18 @@ impl<'a, C: Compartment> RootedJsString<'a, C> {
             None
         }
     }
+
+    pub fn to_property<S>(self, cx: &mut Context<S>) -> Option<PropertyKey<'_, C>>
```

Comment:
Improve naming consistency:
```suggestion
    pub fn to_property_key<S>(self, cx: &mut Context<S>) -> Option<PropertyKey<'_, C>>
```